src/Downloader.ts

```diff
@@ -611,7 +612,14 @@ class Downloader {
     if (isBitmapImageMimeType(resp.headers['content-type'])) {
       if (isWebpCandidateImageMimeType(this.webp, resp.headers['content-type']) &&
           !this.cssDependenceUrls.hasOwnProperty(resp.config.url)) {
-        resp.data = await imagemin.buffer(resp.data, imageminOptions.get('webp').get(resp.headers['content-type']));
+        resp.data = await imagemin.buffer(resp.data, imageminOptions.get('webp').get(resp.headers['content-type']))
+        .catch( async (err) => {
+          if (/Unsupported color conversion request/.test(err.stderr)) {
+            return await imagemin.buffer(await sharp(resp.data).toColorspace('srgb').toBuffer(), imageminOptions.get('webp').get(resp.headers['content-type']));
+          } else {
+            return await imagemin.buffer(resp.data, imageminOptions.get('default').get(resp.headers['content-type']));
```

Comment:
The content-type should be set earlier and if the conversion failed, it should be not set to webp. Unfortunately we have anyway to keep webp extension otherwise we will have broken links.